lib/page.js

```diff
@@ -10,15 +10,25 @@
 // Everything is stored using json flat files.
 
 // #### Requires ####
-const fs = require('fs')
-const path = require('path')
-const events = require('events')
-const glob = require('glob')
-
-const async = require('async')
-
-const random_id = require('./random_id')
-const synopsis = require('wiki-client/lib/synopsis')
+// const fs = require('fs')
+// const path = require('path')
+// const events = require('events')
+// const glob = require('glob')
+//
+// const async = require('async')
+//
+// // const random_id = require('./random_id.cjs')
```

Comment:
is `random_id` not in use?

Reply:
There are several unused imports throughout the files.   As far as I can tell, random_id is not used in pages.